code/game/gamemodes/cult/structures/pylon.dm

```diff
@@ -115,49 +108,49 @@
 //Run each process loop, this function checks if we can stop processing yet.
 //Returns 0 if its time to stop. Returns 1 if we should keep going.
 /obj/structure/cult/pylon/proc/check_process()
-	if (isbroken)
-		return 0
+	if(isbroken)
+		return FALSE
 
-	if (pylonmode == 2)
-		return 1
+	if(pylonmode == 2)
```

Comment:
I know these modes are explained by comments at the top, but it might be nice if they were defines instead?